src/Torann/LaravelMetaTags/MetaTag.php

```diff
@@ -127,7 +127,7 @@ public function tag($key, $value = '')
     {
         return $this->createTag([
             'name' => $key,
-            'content' => $value
+            'content' => $value?:$this->metas[$key]
```

Comment:
This could cause an error in some apps. Try:

```php
'content' => $value ?: Arr::get($this->metas, $key, ''),
```

> Don't Forget to add `use Illuminate\Support\Arr;` to the top of the file.